packages/dropdowns/src/elements/combobox/Listbox.tsx

```diff
@@ -22,6 +22,9 @@ import { ThemeContext } from 'styled-components';
 import { DEFAULT_THEME } from '@zendeskgarden/react-theming';
 import { composeEventHandlers } from '@zendeskgarden/container-utilities';
 
+/**
+ * 1. Hide from NVDA when collapsed to avoid incorrect / missing announcements caused by animation
```

Comment:
[nit] this comment could be inlined – the header notation is more of a thing within CSS view components 😉